src/interop.jl

```diff
@@ -47,7 +47,6 @@ TimeSpans.stop(span::AlignedSpan) = time_from_index(span.sample_rate, span.last_
 # TimeSpan -> AlignedSpan is supported by passing to Intervals
 to_interval(span) = Interval{Nanosecond,Closed,Open}(start(span), stop(span))
 to_interval(span::Interval) = span
-to_interval(span::AlignedSpan) = Interval(span)
```

Comment:
does this count as breaking?

Reply:
No bc this actually errored (no such Interval constructor existed), but removing this dispatch means we hit the fallback which works successfully (as the test shows).